src/types/__snapshots__/resolveDescriptors.spec.ts.snap

```diff
@@ -258,6 +258,16 @@ Line 8, col 21:
 "
 `;
 
+exports[`resolveDescriptors should fail descriptors for case-26 1`] = `
+"<unknown>:13:14: Expected constant value, got A
```

Comment:
This is much closer now, however, I'd expect the `^~~` pointer at `b` and the error message saying that the `b` field is not present in the `A` struct